### PR TITLE
feat: Add mute button to test environment

### LINKED ISSUE #299: Improve the experience for internal testers
Improve the experience for internal testers so that they don't need to listen to all test sound for test the whole test flow including pages after the test. Some proposals are:

1. Click a button to complete the test.
2. Add a mute audio button in the `TestScreen`

### PR DESCRIPTION
**Pull request by dansvend**
- Add mute button in `TestScreen` available in non production environments to improve the experience for internal testers.

Closes #299

### CODE REVIEW

File: screens/TestScreen.tsx [hunk 1]
```diff
@@ -402,7 +412,17 @@ class TestScreen extends Component {
...
+              name={this.state.isVolumeMuted ? "volume-mute" : "volume-high"}
+              onPress={() =>
+                this.setState((prevState) => ({
+                  isVolumeMuted: !prevState.isVolumeMuted,
+                }))
+              }
+              variant="ghost"
+              disabled={isProduction}
+              style={{ opacity: isProduction ? 0 : 1 }}
+            />
```
Comment (tormodAase) [inline screens/TestScreen.tsx:424]: I would put this in a separate component. isVolumeMuted can be a redux state. Then you can create a selector for volume that depends on isVolumeMuted

**Review by tormodAase — CHANGES_REQUESTED** [review]

**Review by tormodAase — APPROVED** [review]

File: screens/TestScreen.tsx (hunk 1 above)
Comment (dansvend) [reply screens/TestScreen.tsx:424]: I'll do this in #310  🙌 Thanks for the feedback 🙏

File: screens/TestScreen.tsx [hunk 2]
```diff
@@ -112,6 +116,13 @@ class TestScreen extends Component {
     this.state.netInfoEventListener();
   }
 
+  getSystemVolume() {
+    if (isProduction) {
+      return SYSTEM_VOLUME;
+    }
```
Comment (tormodAase) [inline screens/TestScreen.tsx:122]: I would remove this part. if isVolumeMuted is true, you should always return 0. However, whether the user is able to mute should be decided elsewhere

File: screens/TestScreen.tsx [hunk 3]
```diff
@@ -401,7 +412,17 @@ class TestScreen extends Component {
...
+              name={this.state.isVolumeMuted ? "volume-mute" : "volume-high"}
+              onPress={() =>
+                this.setState((prevState) => ({
+                  isVolumeMuted: !prevState.isVolumeMuted,
+                }))
+              }
+              variant="ghost"
+              disabled={isProduction}
+              style={{ opacity: isProduction ? 0 : 1 }}
+            />
```
Comment (tormodAase) [inline screens/TestScreen.tsx:425]: I would still separate this into a separate component. That way you can get rid of isProduction at the top of this file

**Review by tormodAase — APPROVED** [review]

File: screens/TestScreen.tsx (hunk 3 above)
Comment (dansvend) [reply screens/TestScreen.tsx:425]: Good point 🙏 I'll keep this in mind in #311 👌

File: screens/TestScreen.tsx (hunk 2 above)
Comment (dansvend) [reply screens/TestScreen.tsx]: Thanks for the feedback 🙏 I removed this part 👌

File: screens/TestScreen.tsx (hunk 3 above)
Comment (dansvend) [reply screens/TestScreen.tsx]: Good point 👌 I moved this into a separate component 📦

**Review by tormodAase — DISMISSED** [review]

**Review by tormodAase — DISMISSED** [review]

**Review by tormodAase — APPROVED** [review]

**Pull request merged**